packages/ethereum-contracts/ops-scripts/deploy-framework.js

```diff
@@ -859,11 +840,9 @@ module.exports = eval(`(${S.toString()})({skipArgv: true})`)(async function (
             "ERC2771_FORWARDER"
         );
 
-        const simpleAclAddress = await getOrDeployHelper(
-            SimpleACL,
-            () => superfluid.getSimpleACL(),
-            "SIMPLE_ACL"
-        );
+        // SimpleACL has now been deployed on all networks.
```

Comment:
Okay. What about deploying to a new network, does the bootstrap logic still exist?